test/runtests.jl

```diff
@@ -49,4 +49,10 @@ end
     a2 = [0. -3.; 4. 0.]
     @test a1 ∈ m
     @test a2 ∉ m
+
+@testset "Interval matrix rand" begin
+    m = IntervalMatrix([-1.1..0.9 -4.1.. -3.9; 3.9..4.1 -1.1..0.9])
+    for i in 1:3
+        rand(m)
```

Comment:
we can now add a check:

```suggestion
        @test rand(m) ∈ m
```
(after rebasing with master)